docs/deployment/kubernetes.md

```diff
@@ -4,9 +4,10 @@ This guide is for deployment to a vanilla Kubernetes 1.8 or 1.9 cluster running
 
 ## Kubernetes
 
-OpenFaaS is Kubernetes-native and uses *Deployments*, *Service*s and *Secret*s. For more detail check out the ["faas-netes" repository](https://github.com/openfaas/faas-netes).
+OpenFaaS is Kubernetes-native and uses *Deployments*, *Services* and *Secrets*. For more detail check out the ["faas-netes" repository](https://github.com/openfaas/faas-netes).
```

Comment:
This is deliberate - Service is an API object, Services is not - same for Secret